thoth/storages/graph/models.py

```diff
@@ -52,6 +52,21 @@ class PackageExtractRun(VertexBase):
     duration = model_property(type=int)
 
 
+@attr.s(slots=True)
+class PackageAnalyzerRun(VertexBase):
```

Comment:
@fridex 1. This is how a new node is created in DGraph, right? Just curious why it's parent class is called VertexBase? 2. Should I just change the name to ```PythonFileDigests``` instead of ```PackageExtractRun``` to sync the results of Package Analyzer?

Reply:
`PythonFileDigests` will point to `PythonFileDigests` so keep this one and create `PythonFileDigests`. The base class implements the actual syncing logic, these are leaf classes which define the model properties.

Reply:
Done